packages/prismic-generator-generator/app/templates/typescript/generators/create-slice/index.ts

```diff
@@ -75,15 +81,24 @@ export default class CreateSlice extends PrismicGenerator {
 
     const description = toDescription(this.answers.sliceName)
 
+    const slicesDirectoryPath = path.join('.slicemachine', 'assets', this.answers.library, this.answers.sliceName)
```

Comment:
can you share this code because you copied this in every single package, sounds about wrong here.

Reply:
That's been done... it's been moved the prismic-yeoman-generator package